src/main/resources/application-dev.yml

```
@@ -23,3 +33,9 @@ auth:
     kakao:
       kakao_grant_type: ${KAKAO_GRANT_TYPE}
       kakao_grant_type_name: ${KAKAO_GRANT_TYPE_NAME}
+
+myapp:
+  sqs:
+    queue-name: ${DEV_AWS_SQS_QUEUE_NAME}
+    queue-url: ${DEV_AWS_SQS_QUEUE_URL}
+    messageDelaySecs: 0
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)